pkgs/by-name/dy/dynamodb-local/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchurl
+, jre
+, makeBinaryWrapper
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "dynamodb-local";
+  version = "2023-12-14";
+
+  src = fetchurl {
+    url = "https://d1ni2b6xgvw0s0.cloudfront.net/v2.x/dynamodb_local_${finalAttrs.version}.tar.gz";
+    hash = "sha256-F9xTcLNAVFVbH7l0FlMuVNoLBrJS/UcHKXTkJh1n40w=";
+  };
+
+  sourceRoot = ".";
+
+  nativeBuildInputs = [ makeBinaryWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin $out/share/dynamo-db-local
+    cp -r DynamoDBLocal* $out/share/dynamo-db-local
+
+    makeBinaryWrapper ${jre}/bin/java $out/bin/dynamo-db-local \
```

Comment:
How about keeping the same name as in `pname` ?

Basically, replacing `dynamo-db-local` by `dynamodb-local`.

Reply:
makes sense 👍